### DISCUSSION TITLE
dockerfile missing

### DISCUSSION THREAD

**Discussion by vincensiusadi** [etf1/kafka-message-scheduler#23] [Q&A]
how to build this with dockerfile?

**Comment by fkarakas** [comment]
Hello @vincensiusadi,
The docker file is in folder `./cmd/kafka/Dockerfile`, but i just added a build target in the Makefile, so you can build the image with the command `make docker`.
You have an available image in docker hub `etf1/kafka-message-scheduler` if you want to use it directly.
Regards
Fatih
